jc/parsers/traceroute.py

```diff
@@ -164,7 +165,7 @@ class info():
 SOFTWARE.
 '''
 
-RE_HEADER = re.compile(r'(\S+)\s+\((\d+\.\d+\.\d+\.\d+|[0-9a-fA-F:]+)\)')
+RE_HEADER = re.compile(r'traceroute6? to (\S+)\s+\((\d+\.\d+\.\d+\.\d+|[0-9a-fA-F:]+)\)')
```

Comment:
In the streaming parser, we need to detect header without without relying on the line index. This requires a more specific patten that  matches only traceroute headers.